core/lib/zksync_core/src/lib.rs

```diff
@@ -505,6 +506,35 @@ pub async fn initialize_components(
         tracing::info!("initialized State Keeper in {elapsed:?}");
     }
 
+    if let Some(cfg) = configs.consensus_config.clone() {
```

Comment:
Would it make sense to create a `consensus` component and exit with an error if such a component is requested, but config is not provided?

Reply:
Consensus is in practice a part of state keeper, but I can add it as a separate component if you want. Should I do so?

Reply:
Up to you. Personally, I think I'd prefer it to be explicitly stated whether we want to run consensus or not.
E.g. right now if you add the required variables to the gitops-kubernetes, it'll immediately restart with consensus enabled, and in order to disable it, if we ever want to, you'll have to remove all these variables.
With a component-based approach enabling and disabling will be handled by adding a CLI argument independently of whether the config values are present.